workers/unity/Packages/com.improbable.gdk.core/Systems/WorldCommandsSendSystem.cs

```diff
@@ -64,7 +65,7 @@ protected override void OnCreateManager(int capacity)
 
         protected override void OnUpdate()
         {
-            if (worker.Connection == null)
+            if (connection == null)
```

Comment:
is this even needed? When creating a worker, a connection has to be present.

Reply:
No idea. This is just me fixing up the code that depends on this. I didn't add this check.

Reply:
Can add it to the drive by list.